+sig/playgroundPTB.m

```diff
@@ -19,17 +19,17 @@
   'TimerFcn', @process, 'Name', 'MainLoop');
 cp = hw.CursorPosition;
 
-vbox = uiextras.VBox('Parent', parent);
+vbox = uix.VBox('Parent', parent);
 
 % Get number of available screens
 nScreens = Screen('Screens');
-screenNum = iff(max(nScreens) > 2, 1, 0);
+screenNum = iff(nScreens(end) > 1, 1, 0);
```

Comment:
Out of interest why did you change `max(nScreens)` to `nScreens(end)`?  It seems slightly less readable to me.

Reply:
just because it doesn't require any computation on the array. obviously very minor. if you'd rather have it as 'max' because you think it's more readable I can change it back.

Reply:
Again I don't really care but for things like this the difference in processing time is literally incalculable so I would prefer readability over potential performance.